src/drawing-tool/components/take-snapshot.tsx

```diff
@@ -45,7 +45,7 @@ export const TakeSnapshot: React.FC<IProps> = ({ authoredState, interactiveState
     <>
       {
         snapshotTarget &&
-        <button className={cssHelpers.apButton} onClick={handleSnapshot} disabled={snapshotInProgress} data-test="snapshot-btn">
+        <button className={cssHelpers.apButton} onClick={handleSnapshot} disabled={snapshotInProgress} data-testid="snapshot-btn">
```

Comment:
@tejal-shah is this going to break any integration tests? 
This is the snapshot button used in the image question  interactive, drawing tool interactive, and labbook interactive.